plugins/woocommerce-admin/client/navigation/style.scss

```diff
@@ -21,6 +21,26 @@
 	.components-navigation__menu-title {
 		overflow: visible;
 	}
+
+	.components-navigation__menu {
+		&::-webkit-scrollbar-thumb {
```

Comment:
I haven't really noticed this as an issue in Firefox, but it is possible to apply some level of [these styles there as well ](https://stackoverflow.com/questions/6165472/custom-css-scrollbar-for-firefox)(but not via the webkit vendor pseudo-selectors).

Reply:
@joelclimbsthings Good call.  Added those in just in case in https://github.com/woocommerce/woocommerce/pull/33313/commits/9f46fbf165371faf5b39722b5336be24813f2ad6

Reply:
The values specified were not valid. I've addressed this in commit 93b6115bb2